pika/heartbeat.py

```diff
@@ -8,38 +8,37 @@
 
 
 class HeartbeatChecker(object):
-    """Checks to make sure that our heartbeat is received at the expected
-    intervals.
+    """Sends heartbeats to the broker and checks to make sure that the broker's
+    heartbeat is received before the expected timeout expires.
 
     """
-    DEFAULT_INTERVAL = 60
-    MAX_IDLE_COUNT = 2
-
+    _MAX_IDLE_COUNT = 2
     _STALE_CONNECTION = "Too Many Missed Heartbeats, No reply in %i seconds"
 
-    def __init__(self, connection, interval=DEFAULT_INTERVAL, idle_count=MAX_IDLE_COUNT):
-        """Create a heartbeat on connection sending a heartbeat frame every
-        interval seconds.
+    def __init__(self, connection, interval):
```

Comment:
@lukebakken - help, I am confused! :) We have been discussing this for some time and agreed to rename the arg interval to timeout, and it's now back to interval. Interval is meaningless here because we shouldn't use it as interval between anything, and timeout has the correct semantics.

Reply:
Maybe we have a different understanding of an interval. To me, an "interval of 60 seconds" means that every 60 seconds something happens, which describes what we do - send and check heartbeats (using our differently-computed values). Is that not your understanding of "interval"?

Reply:
RabbitMQ server will treat the negotiated period of time as a timeout (2 intervals), regardless of what it's named in the client.

Reply:
Thanks @michaelklishin I will use the same logic here. Two "heartbeat check" intervals without a heartbeat from the server or activity on the channel will be considered a timeout.